frontend/src/deployment/metadata.ts

```diff
@@ -117,7 +117,12 @@ function getRuleMetadata(srcDir: string, language?: string) {
     }
     const languageFile = path.join(srcDir, language, 'metadata.json');
     if (fs.existsSync(languageFile)) {
-      return JSON.parse(fs.readFileSync(languageFile, 'utf8'));
+      try {
+        return JSON.parse(fs.readFileSync(languageFile, 'utf8'));
+      } catch (error) {
+        console.error("When parsing the JSON file " + languageFile);
```

Comment:
```suggestion
        console.error('When parsing the JSON file ' + languageFile);
```
That should address the code smell.

But maybe it's also worth printing the exception? I'm not sure how it will be displayed at the moment.

Reply:
The error will be printed together with the stacktrace just below this, because it is rethrown.
This change address the issue that when this exception happens it does not say which file has broken syntax.